File: VB/MCAuth/json/selectedProfile.vb
Imports System.Runtime.Serialization

Namespace json
    <DataContract> _
    Class selectedProfile
        <DataMember(Order:=0, IsRequired:=False)> _
        Public name As String = ""
        <DataMember(Order:=1, IsRequired:=False)> _
        Public id As String = ""
    End Class
End Namespace

File: C#/MCAuth/json/selectedProfile.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;

namespace MCAuth.json
{
    [DataContract]
    class selectedProfile
    {
        [DataMember(Order = 0, IsRequired = false)]
        public string name = "";
        [DataMember(Order = 1, IsRequired = false)]
        public string id = "";
    }
}
